app/controllers/live_controller.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+class LiveController < ApplicationController
+  def admin
+    @competition_id = params[:competition_id]
+    @competition = Competition.find(@competition_id)
+    @round = Round.find(params[:round_id])
+    @event_id = @round.event.id
+    @competitors = @round.registrations_with_wcif_id
+  end
+
+  def add_result
+    results = params.require(:attempts)
+    round_id = params.require(:round_id)
+    registration_id = params.require(:registration_id)
+
+    if LiveResult.exists?(round_id: round_id, registration_id: registration_id)
+      return render json: { status: "result already exist" }, status: :unprocessable_entity
+    end
+
+    AddLiveResultJob.perform_now({ results: results,
+                                   round_id: round_id,
+                                   registration_id: registration_id,
+                                   entered_by: current_user })
```

Comment:
Regarding my other comment at https://github.com/thewca/worldcubeassociation.org/pull/10685#discussion_r1944347347: You would also need to update the call here and get rid of the hash `{` braces.